### PR TITLE
Update install.md to reflect latest Cats Effect 3 version (3.2.4)

### PR DESCRIPTION
**Pull request by ajmanor**
Thanks for opening a PR!

Target the `main` branch if this PR is targeted for the 3.x series and the `series/2.5.x` branch if targeted for the 2.5.x series.

If the CI build fails due to Scalafmt, run `sbt fmt` and push the changes (generally a good idea to run this prior to opening the PR).

Feel free to ask questions on the fs2 and fs2-dev channels on the [Typelevel Discord server](https://discord.gg/9V8FZTVZ9R).

### CODE REVIEW

File: site/getstarted/install.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Install
 
-The latest version for Cats Effect 3 is `3.2.0`, which supports Cats Effect 3 and is cross built for Scala 2.12, 2.13, and 3.0.
+The latest version for Cats Effect 3 is `3.2.4`, which supports Cats Effect 3 and is cross built for Scala 2.12, 2.13, and 3.0.
```
Comment (armanbilge) [inline site/getstarted/install.md:3]: Actually :) https://github.com/typelevel/cats-effect/releases/tag/v3.3.5

```suggestion
The latest version for Cats Effect 3 is `3.3.5`, which supports Cats Effect 3 and is cross built for Scala 2.12, 2.13, and 3.0.
```

We could set this automatically with mdoc btw.

Comment (armanbilge) [reply site/getstarted/install.md:3]: Woops sorry, ignore me

Comment (mpilquist) [reply site/getstarted/install.md:3]: Wait, how?

Comment (armanbilge) [reply site/getstarted/install.md:3]: Here's how I do it in the sbt-typelevel-site plugin:
https://github.com/typelevel/sbt-typelevel/blob/3e343113edd5ae1153505efd7feff57579b51cfa/site/src/main/scala/org/typelevel/sbt/TypelevelSitePlugin.scala#L64-L70

https://github.com/typelevel/sbt-typelevel/blob/3e343113edd5ae1153505efd7feff57579b51cfa/docs/index.md?plain=1#L14

Actually, I can extract those settings out into a `TypelevelMdocPlugin` so you can get those variables for free without buying into the rest of the sbt-typelevel-site stuff.

**Comment by mpilquist** [comment]
Thanks! Closing since this was fixed in #2819 via template expansion.

**Pull request closed without merging**